server/lib/notifications/email.ts

```diff
@@ -133,10 +133,14 @@ export const notifyByEmail = async (activity: Activity) => {
     case ActivityTypes.COLLECTIVE_EXPENSE_CREATED:
     case ActivityTypes.COLLECTIVE_FROZEN:
     case ActivityTypes.COLLECTIVE_UNFROZEN:
-    case ActivityTypes.COLLECTIVE_UNHOSTED:
     case ActivityTypes.PAYMENT_CREDITCARD_EXPIRING:
       await notify.collective(activity);
       break;
+    case ActivityTypes.COLLECTIVE_UNHOSTED:
+      await notify.collective(activity, {
+        replyTo: 'support@opencollective.com',
```

Comment:
Good candidate for the reply to email introduced in https://github.com/opencollective/opencollective-api/pull/8006:
```suggestion
        replyTo: activity.data.host.data?.replyToEmail || 'support@opencollective.com',
```